README.md

```diff
@@ -7,54 +7,7 @@ Package license: MIT
 
 Feedstock license: BSD 3-Clause
 
-Summary: PyPy is a Python interpreter and just-in-time compiler. This is a Python3.6 compatible version.
-
-
-How to use this package?
-========================
-
-Since PyPy is an alternative *interpreter*, using this package is different than other conda-forge packages.
-
-Note that currently this package only works for macOS and Linux.
-
-You want to create an **empty** Conda environment:
-
-```
-conda create -n pypyenv
-conda activate pypyenv
-```
-
-(🔴️ you might still have `python` or `pip` binaries in your $PATH, but those are the system's and you MUST NOT use them!️)
-
-Then, install this package:
-
-```
-conda install -c conda-forge pypy3.6
-```
-
-Now you can install `pip`:
-
-```
-pypy3 -m ensurepip
-```
-
-🎉 using this `pip` you can install any package you like! 
-For example, to update `pip` itself (which you should do!):
-
-```
-pypy3 -m pip install -U pip
-```
-
-or
-
-```
-pip3 install ...
-```
-
-(You can verify that it's the correct `pip3`/`pip` using `which {pip3,pip}`. It should be something like `.../miniconda3/envs/pypyenv/bin/pip` )
-
-🔴 You CANNOT use `conda install x` to install anything,
-because Conda provides pre-built package for the regular CPython interpreter, which are incompatible with PyPy.
+Summary: PyPy is a Python interpreter and just-in-time compiler.
```

Comment:
Don’t let the rerender edit this part of the file :)

Reply:
That is kind of against what we expect of the rerender. Maybe we should document that elsewhere that won't be overwritten automatically.

Reply:
The text there is not true anymore. See https://conda-forge.org/blog//2020/03/10/pypy

Reply:
Awesome! I guess we don’t need this text anymore!